fingerprinting-protections/init.js

```diff
@@ -25,7 +24,6 @@ function init(window) {
             callback(0, 9999999999)
         }
     };
-}
 
 if (typeof module !== 'undefined' && module.exports) {
     module.exports = init
```

Comment:
You've removed this method so this will error.

This changes the test a lot and I'm not that familiar with how it's imported into other platforms but I suspect however it's imported we should at a minimum delete this line.

I'm guessing this is a common JS issue and realistically we should be making these run as ESM modules, however you might need to use common JS in your testing code.

I'd defer to @kdzwinel who I think wrote these though.

Reply:
I've updated the PR to restore the init function. The only difference is the platform exceptions for iOS and macOS